packages/angular_devkit/build_angular/src/angular-cli-files/utilities/bundle-calculator.ts

```diff
@@ -5,22 +5,109 @@
  * Use of this source code is governed by an MIT-style license that can be
  * found in the LICENSE file at https://angular.io/license
  */
+import * as webpack from 'webpack';
+import { ProcessBundleFile, ProcessBundleResult } from '../../../src/utils/process-bundle';
 import { Budget } from '../../browser/schema';
 
-export interface Compilation {
-  assets: { [name: string]: { size: () => number } };
-  chunks: { name: string, files: string[], isOnlyInitial: () => boolean }[];
-  warnings: string[];
-  errors: string[];
-}
-
 export interface Size {
   size: number;
   label?: string;
 }
 
-export function calculateSizes(budget: Budget, compilation: Compilation): Size[] {
-  const calculatorMap = {
+export interface Threshold {
+  limit: number;
+  type: ThresholdType;
+  severity: ThresholdSeverity;
+}
+
+export enum ThresholdType {
+  MAX = 'maximum',
```

Comment:
nit: Pascal case is typically used for enums.  All uppercase is generally reserved for module level constants.

Reply:
Done.